pki/interface.go

```diff
@@ -48,6 +72,10 @@ type Validator interface {
 	// AddTruststore adds all CAs to the truststore for validation of CRL signatures. It also adds all CRL Distribution Endpoints found in the chain.
 	// CRL Distribution Points encountered during operation, such as on end user certificates, are only added to the monitored CRLs if their issuer is in the truststore.
 	AddTruststore(chain []*x509.Certificate) error
+
+	// SubscribeDenied registers a callback that is triggered everytime the denylist is updated.
+	// This can be used to revalidate all certificates on long-lasting connections by calling Validate on them again.
+	SubscribeDenied(f func())
```

Comment:
`SubscribeUpdated`?

Reply:
You only subscribe to updates on the denylist, so I wanted to have something indicating that in the name.